package.json

```diff
@@ -21,7 +21,8 @@
     "browser:dev": "tsx browser/src/index.ts",
     "prebrowser:build": "npm install",
     "browser:build": "tsc --project browser",
-    "browser:start": "node browser/dist/index.js"
+    "browser:start": "node browser/dist/index.js",
+    "healthcheck": "npx tsx scripts/healthcheck.ts"
```

Comment:
Update the create agent script to call this instead of the current screen -ls call